spec/kolekti_analizo/collector_spec.rb

```diff
@@ -57,10 +62,14 @@
     end
 
     describe 'default_value_from' do
-      let(:metric_list) { FactoryGirl.build(:analizo_metric_collector_list).raw }
+      let!(:metric_list) { FactoryGirl.build(:analizo_metric_collector_list).raw }
```

Comment:
Is there a way to mock the `load_analizo_supported_metrics` method or even the `metric_list` method so we do not need all this "low level" mocks? The way things are, our tests are too coupled with production code.

We can see this on this very commit, where changing one method force us to change another two tests.